### PR TITLE
Add email for business projects and new dashboard change

### PR DESCRIPTION
**Pull request by agjohnson**
This might not be how we've sent some of the last emails, but hte copy
should be reusable.

- Fixes https://github.com/readthedocs/meta/issues/163

### CODE REVIEW

**Comment by humitos** [comment]
> This might not be how we've sent some of the last emails

Yeah, we are not sending emails using the application for these type of emails. Now, we do the following:

1. Write the email in HackMD (example: [Drop support for Subversion, Mercurial and Bazaar](https://hackmd.io/ZiEI9rbTSI651i71pPyxbw))
2. Use the first line for the subject of the email
3. Make the document _public_
4. Use [a script like this one](https://gist.github.com/humitos/f0fd509a4256cf54f6529571c66de07c) to send the email from production instance

**Review by humitos — COMMENTED** [review]
Content looks good to me -- Just noted in another comment that we don't need to put this code into the application and use HackMD and a small script to send this email 👍🏼

File: readthedocs/templates/projects/notifications/new_dashboard.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    </a>
+  </p>
+
+  <hr />
+
+  <p>
+    On <strong>Dec 10th</strong> we will make our new dashboard the default
+    dashboard for users logging into <a href="https://readthedocs.com">Read the Docs Business</a>
+    from our website. This change will mainly affect project maintainers,
+    especially users that have not yet switched to the new dashboard.
```
Comment (ericholscher) [inline readthedocs/templates/projects/notifications/new_dashboard.html:29]: maintainers and users here is confusing.

```suggestion
    from our website. This change will mainly affect project maintainers,
    especially maintainers that have not yet switched to the new dashboard.
```

**Review by ericholscher — COMMENTED** [review]
Content looks solid otherwise.

**Comment by agjohnson** [comment]
> Yeah, we are not sending emails using the application for these type of emails.

Makes sense. This process should be formalized and documented.

**Comment by agjohnson** [comment]
> Yeah, we are not sending emails using the application for these type of emails.

Makes sense, this needs to be formalized and documented. I think it would make sense as a repo instead of HackMd though, or at least these emails should all be stored in Git after copy is improved in HackMD. It's not great that we can't find or review previous emails.

**Comment by agjohnson** [comment]
Actually, looking at the script you linked, I'm not so sure on this piece either. What is the benefit to using a one-off script? It feels like that script is mostly duplicating what the email notification class already does.

It's feeling easier to just continue with this PR, I'll likely just use this process for this email.

**Comment by agjohnson** [comment]
I've sent out tests on both platforms.

**Comment by humitos** [comment]
> What is the benefit to using a one-off script?

- No need to write HTML _and_ TXT files
- Content written in Markdown
- Easier to run without deploying the application
- Some scripts contains private information we don't want to expose publicly
- No need to maintain a bunch of Python code for a one-time email

**Comment by humitos** [comment]
This is added in the release roadmap to be merged before building the AMIs. So, I'm merging it since I think you will need this code in production to send the email to customers.

**Pull request merged**